lib/api/completeMultipartUpload.js

```diff
@@ -210,8 +210,13 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 // return an error
                 const partSize = Number.parseInt(storedPartsAsObjects[i]
                     .size, 10);
-                if (i < jsonList.Part.length - 1 &&
+                // allow smaller parts for testing
+                log.info('MPU_TESTING env variable setting',
```

Comment:
You can write this log entry only when `process.env.MPU_TESTING === 'yes'` and skip otherwise. It could throw people off when looking at logs in production.